bibliopixel/drivers/SPI/APA102.py

```diff
@@ -4,7 +4,12 @@
 
 
 class APA102(SPIBase):
-    """Driver for APA102/SK9822 based LED strips on devices like the Raspberry Pi and BeagleBone"""
+    """Driver for APA102/SK9822 based LED strips on devices like
+    the Raspberry Pi and BeagleBone
+
+    Provides the same parameters as
```

Comment:
"parameters" - do we use that word consistently?  Is it defined somewhere?

Ah, "parameter" turns out to be the official Python term for an argument to any callable!  Cool.

What about "constructor parameters"?  It makes it really clear what's going on.